### ISSUE TITLE
BB#48 på kakan av #47a

### ISSUE THREAD

**Issue by Bbbolaget-karl** [sweco-seukni/Bbbolaget#27]
Jag tänkte att vi kör BB#48 på kakan av #25. (ska vara 47 men github mixtrar) nåja. Nu pallade inte wlp500 trycket så vi har kvar att jobba med wlp570, golden ale, vilket känns på sätt och vis lika bra det. 

http://www.whitelabs.com/yeast/wlp570-belgian-golden-ale-yeast

Jag hade först tänkt mig en mer mörk svagare belgare. Har hittat recept med apelsinskal och koriander, om vi nu vill bryta den barriären att tillsätta annat. Men, sen tänkte jag. Man brukar brygga starkt på kakan och i torsdags testade Ulrik och jag kvalitetstid, alltså BB#33. Den var grymt god nu! Alltså. Kanske ska man göra en sådan nu igen. 

Angående jästen så attanuerar den ut bra. Jäser ut hårt, vilket tjänar våra syften bra. men den verkar också ta tid på sig, se länk, vilket nog gör det lagom att brygga #48 på måndag om 8 dagar. 

Den som fick us-05 i sig är ju inte intressant att göra kaka-jäsning på. 

alltså, förslag
6 kg pilsnermalt
koka 90 min
humle är saaz, tettnang och ekg eller så
tillsätta 400 g socker när jäsning börjar avta. 
Lagra i 8-10 månader innan drickning. fin till vårsolen i april 2016!

säger ni?

**Comment by sweco-seukni** [comment]
I vote in favor, mycket mindre socker nu än sist, får vi upp den till 9% ändå? Inte för att vi måste utan mest nyfiket.

Vi kan labba lite med humlegivorna på plats, såg att det var Styrian Goldings i orginalreceptet, vi måste testa det snart, lägher upp en lapp i Trello

**Comment by Bbbolaget-karl** [comment]
Skoj! Då blir det så. 
Angående om vi får up den till 9% så har det rätt mycket att göra med mäskeffektiviteten, tänker jag. Som det ser ut nu så får andra på forumet ut OG 1070 på 6 kg malt när vi får 1060. Håller på o funderar på vad vi gör galet, gissning är att vi ska laka mer, finns några andra bubblare också. inget klartänkt, tyvärr. 

Gissar att vi ska upp till ca 1078 eller så får att nå de där 9%. 

glömde beställa styrian goldings, hinner inte till mån nästa vecka tyvärr. Du kan ju om du vill kolla förbi bryggaffären. Ok ändå?

**Comment by Bbbolaget-karl** [comment]
Tidigare var med 4,8 kg malt och 0,7 kg socker. Det är ungefär 30% socker. För mycket. jag vill ner till  10% och tillsätta sockret efter några dagars jäsning. 
Alltså, 6 kg pilsnermalt, vi får koka i 90 min. 
Verkar som lakningen gör att vi hamnar lågt i vårt utbyte. nåja

60% av 6 kg är 3,6 kg maltsocker ut, alltså inte mer än 360 g strösocker ner i batchen.

**Comment by Bbbolaget-karl** [comment]
45IBU fördelade på
25@60
20@30
Då hade vi EKG och Tettnang. 

Nu tror jag vi tar Saaz @30 och nån hallertau @60.

**Comment by Bbbolaget-karl** [comment]
Utbytet beror också på mäskningstiden. Nu bryter vi ju den med en utmäskningsrast. Testar att förlänga till 90 min. @66 °C alltså. Spelar ju inte nån roll eftersom den kan stå på på eftermiddagen ändå.

**Comment by Bbbolaget-karl** [comment]
Jäser väldigt bra. Satte igång redan efter några timmar. 
Har kalkylerat lite vilket sockertillskott 340 g socker gör till våra 18 liter i tunnan. Det kommer höja det fiktiva OGt från 1070 till 1077 (eftersom vi tillsätter sockret senare när SG är betydligt under 1070). Jäser vi ut till 1010 hamnar vi på vad vi var förra gången, ca 9%abv. 
Har också kollat in shbf:s typdeffar. 
Följande gäller för Kategori 9D, ljus stark belgisk ale
OG 1.067 –
FG 1.006 – 1.016
Alkohol (vol%) 7.0 – 13.0
IBU 20 – 35
Färg (°EBC) 7 – 30 

Samtliga ser ut att vara ok utom IBU där vi ligger 10 högt. Men det kan bli så att den tonas ner till provtillfället om vi lagrar den nu. Tänker jag. Dessutom upp till domarna att pricka det.

**Comment by sweco-seukni** [comment]
Kommer vi inte jäsa lägre än 1010? Det var väl 1001 sist, har vi någon koll på hur jästen reagerar med mindre socker? Kanske ska ta ännu mindre socker för att se lite på ytterligheterna, så vikan lägga oss på en lagom nivå när vi kör den skarpa sm-batchen.
Om vi kör 200g och räknarmed att vi jäser ner till 1006 borde vi hamna på ca 9abv. Vad sägs, tappar vi något annat på att ha mindre socker?

**Comment by Bbbolaget-karl** [comment]
Jo, det förstås. Du har en bra poäng där. Förutsättningarna var lite annorlunda då, dock
Vi hade bortåt 30% av det jäsbara innehållet som socker (nu ca 10%). Att tillsätta socker i efterhand, den process vi nu gör, skapar förutsättningar för att jäsa ut hårdare också, totalt sett. Annan jäst då, wlp545 jäste till 1001 och wlp 550 till 1006. 

Vi har ju en liten fingervisning också med IPAn, eftersom vi tog ett SG på den. den var ju på 1013. Men ok, där tillsätter vi ju inte något socker. 

Du har nog rätt att vi testar denna gång och sen gör en till, till skarp SM, men vi kan ju lägga 5 av denna på sm-flaska ändå, så får vi blindtesta nån gång. skoj =)

Räknat om. Tillsätter vi 200 g socker ger det 4,5 öschlepoäng, eller adderat till 1074. Jäser den ut till 1006 är vi på 8,9%.

När man tillsätter socker gör man ofta det för att få upp abv utan att få upp maltkroppen, att man vill balansera tex alkoholsmaken mot jästsmaken. Eller vad det nu är. Här tänker tänker jag att det mer än väl räcker med de 9%en

Däremot att inte tillsätta socker alls, ger 1070-1013, 7,5%. kanske inte lågt, men för mycket restsötma, tänker jag. vill ha den mer stram. 

säger du?

**Comment by sweco-seukni** [comment]
Just de, annan jäst nu, mycket som spelar in, håller med om att vi vill ha den lite stram så vi bör ha med socker, tycker vi kör med 200g!

**Comment by Bbbolaget-karl** [comment]
200 gram är det sagt, å så blir det!

**Comment by Bbbolaget-karl** [comment]
26/6 - SG - 1023
27/6 jäsbubbla i röret ungefär var 5te till var 10nde sekund. Tillsätter 200 g socker i lösning (uppkokad m 200 g vatten)

**Comment by Bbbolaget-karl** [comment]
FG 1009
denna döps om till BB#51 i etikettsammanhang. garvarna eller kvalitetstid.

**Comment by Bbbolaget-karl** [comment]
Garvarna blev det. 
8,5% vol
